spk/nzbhydra/Makefile

```diff
@@ -0,0 +1,48 @@
+SPK_NAME = nzbhydra
+SPK_VERS = 0.0.1a78
+SPK_REV = 1
+SPK_ICON = src/nzbhydra.png
+DSM_UI_DIR = app
+
+DEPENDS  = 
+SPK_DEPENDS = "python>2.7.9:git"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = NZBHydra is a meta search for NZB indexers and the \"spiritual successor\" to NZBmegasearcH. It provides easy access to a number of raw and newznab based indexers.
+ADMIN_PORT = 5075
+RELOAD_UI = yes
+DISPLAY_NAME = NZBHydra
+BETA = 1
+CHANGELOG = "1. First release"
```

Comment:
If there's one entry in the CHANGELOG, `CHANGELOG = First release` is sufficient.